package.json

```diff
@@ -29,10 +29,8 @@
     "eslint-config-standard-jsx": "11.0.0",
     "eslint-plugin-import": "2.26.0",
     "eslint-plugin-n": "15.2.3",
-    "eslint-plugin-node": "11.1.0",
```

Comment:
@khamui do we want to enable these others too or no?

Reply:
its the same as `eslint-plugin-n`, that was forked because `...-node` is not maintained anymore

Reply:
ah, yes added back and activated with the latest commit :)